test/cmocka/src/audio/volume/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmocka_test(volume_process
 )
 
 target_include_directories(volume_process PRIVATE ${PROJECT_SOURCE_DIR}/src/audio)
+target_include_directories(sof_options INTERFACE ${PROJECT_SOURCE_DIR}/src/platform/intel/cavs/include)
```

Comment:
This would change include directories for all targets that depend on sof_options. I guess you want something like `target_include_directories(volume_process PRIVATE ...` or `include_directories(...)`

Reply:
I've moved:
`if(CONFIG_CAVS)
	target_include_directories(sof_options INTERFACE ${PROJECT_SOURCE_DIR}/src/platform/intel/cavs/include)
endif()`
to `test/cmocka/CMakeList.txt`, since it is required by more than only volume tests.

Reply:
@jajanusz Are you OK with the new version?